<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>projectAddDialog</class>
 <widget class="QDialog" name="projectAddDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="projectLabel">
       <property name="text">
        <string>Project Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="projectNameEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="addProjectButton">
       <property name="text">
        <string>add project</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="actionLabel">
       <property name="text">
        <string>Action Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="actionLineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="addActionButton">
       <property name="text">
        <string>add action</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeView" name="projectTreeView">
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>addProjectButton</sender>
   <signal>clicked()</signal>
   <receiver>projectAddDialog</receiver>
   <slot>addProject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>4</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addActionButton</sender>
   <signal>clicked()</signal>
   <receiver>projectAddDialog</receiver>
   <slot>addAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>383</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>41</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addProject()</slot>
  <slot>addAction()</slot>
 </slots>
</ui>
